piptools/utils.py

```diff
@@ -11,7 +11,7 @@
 from pip._vendor.packaging.version import parse as parse_version
 from six.moves import shlex_quote
 
-from ._compat import install_req_from_line
+from ._compat import InstallCommand, install_req_from_line
```

Comment:
I thought so but, InstallCommand uses in test also.

Reply:
But we can adjust the test somehow (or remove).

Reply:
Yes, I think `test_create_install_command` shoud be adjusted to test for some property on the result of `create_install_command` instead for example.  It could still do the (real?) import, based on the pip version - but that should not really be ncessary.

Reply:
e.g. `assert install_command.name == 'install'`?

Reply:
Fixed!

Reply:
Looks good, but CI fails (have not checked it).

Reply:
Because `do_import` does some logic to cover importing from ancient versions, old versions and new versions of PIP.

Reply:
The choices are:
1) leave do_import
2) add one more branch `if PIP_VERSION < (10, ): from pip.commands.install import InstallCommand`.

Reply:
I'd prefer 2 - but have not really investigated.
I understand it like 2 would improve the code, while 1 helps the tests?

Reply:
> I understand it like 2 would improve the code, while 1 helps the tests?

IMO the `do_import` makes code less, eg:
```
InstallCommand = do_import("install.commands", "InstallCommand")
```
vs

```
if PIP_VERSION < (10, ):
    from pip.install.commands import InstallCommand
elif PIP_VERSION < (19, 3):
    from pip._internal.install.commands import InstallCommand
```

it just wraps these branches in one function call. I think it would be inconsistent to use the 2nd option only for InstallCommand and do not unwrap the other `piptools.pip_compat.*` stuff, such as PackageFinder, FormatControl, Wheel, etc.

Reply:
Okay, let's go for the second option.

Reply:
Sorry for not replying earlier.  See https://github.com/jazzband/pip-tools/pull/864/files#r312900399.